src/components/domain/BeaconUseSection.tsx

```diff
@@ -0,0 +1,327 @@
+          <BeaconUseDataRowItem label="24-bit HEX" value={use.hexAddress} />
+        )}
+        {use.cnOrMsnNumber && (
+          <BeaconUseDataRowItem
+            label="CORE/Serial number"
+            value={use.cnOrMsnNumber}
+          />
+        )}
+        {use.dongle.includes("true") && (
+          <BeaconUseDataRowItem label="Is this a dongle?" value="Yes" />
+        )}
+        {use.beaconPosition && (
+          <BeaconUseDataRowItem
+            label="Beacon position"
+            value={use.beaconPosition}
+          />
+        )}
+
+        {use.driving.includes("true") && (
+          <BeaconUseDataRowItem value="This beacon is used while driving" />
+        )}
+
+        {use.cycling.includes("true") && (
+          <BeaconUseDataRowItem value="This beacon is used while cycling" />
+        )}
+
+        {use.climbingMountaineering.includes("true") && (
+          <BeaconUseDataRowItem value="This beacon is used while climbing and/or mountaineering" />
+        )}
+
+        {use.skiing.includes("true") && (
+          <BeaconUseDataRowItem value="This beacon is used while skiing" />
+        )}
+
+        {use.walkingHiking.includes("true") && (
+          <BeaconUseDataRowItem value="This beacon is used while walking and/or hiking" />
+        )}
+
+        {use.workingRemotely.includes("true") && (
+          <>
+            <BeaconUseDataRowItem value="This beacon is used while working remotely" />
+            <BeaconUseDataRowItem
+              label="Working remotely location"
+              value={use.workingRemotelyLocation}
+            />
+            <BeaconUseDataRowItem
+              label="Number of people working remotely"
+              value={use.workingRemotelyPeopleCount}
+            />
+          </>
+        )}
+
+        {use.windfarm.includes("true") && (
+          <>
+            <BeaconUseDataRowItem value="This beacon is used at a windfarm or windfarms" />
+            <BeaconUseDataRowItem
+              label="Windfarm location:"
+              value={use.windfarmLocation}
+            />
+            <BeaconUseDataRowItem
+              label="Number of people usually at the windfarm or windfarms:"
+              value={use.windfarmPeopleCount}
+            />
+          </>
+        )}
+
+        {use.otherActivityText && (
+          <>
+            <BeaconUseDataRowItem
+              label="Other activity"
+              value={use.otherActivityText}
+            />
+            <BeaconUseDataRowItem
+              label="Other activity location"
+              value={use.otherActivityLocation}
+            />
+            <BeaconUseDataRowItem
+              label="Number of people present during other activity"
+              value={use.otherActivityPeopleCount}
+            />
+          </>
+        )}
+      </SummaryListItem>
+    </SummaryList>
+  </>
+);
+
+const Communications: FunctionComponent<{ use: BeaconUse }> = ({
+  use,
+}: {
+  use: BeaconUse;
+}): JSX.Element => {
```

Comment:
I've used inline typing here to avoid cluttering the dependency graph with use-once interfaces.  Thoughts?